build.sh

```diff
@@ -69,10 +69,20 @@ if [ x$PULLREQUEST = "xtrue" ]; then
   fi
 fi
 
-# Build steps for SDK libraries:
+# Verify SDK CLI, run unit tests
+_notify_github pending "SDK CLI build running"
+pushd $REPO_ROOT_DIR/cli
+go test ./...
+if [ $? -ne 0 ]; then
+  _notify_github failure "SDK CLI build failed"
```

Comment:
This message seems misleading (build worked, test failed)? But it doesn't matter since we're not relying on _notify_github anymore.